etc/init.d/kano-reboot-clear

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
```

Comment:
Added. NB, I've checked that systemd doesn't care where the LSB block is in the file (so it doesn't matter that the copyright is above it) but this shouldn't be merged back to wheezy without doing the same check in sysvinit.